Rovers/Rovers.tex

```diff
@@ -0,0 +1,387 @@
+This project exists to get our members experience with rovers by building such a rover. 
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+The goal of this project is delivering a functioning mock rover.
+We define mock rover as a functioning rover built to specifications for a mock mission as it will not be launched. 
+We define rover as a vehicle for driving over rough terrain, especially one driven by remote control.
+The rover will allow the team to develop skills and knowledge to build a
+more feature completer rover in the future. 
+These skills include rover fabrication and design, autonomous navigation, electrical system design, rover testing, computer vision, and robotics.
+The rover will be testing using events designed like the URC but scaled down to fit the scope of this rover. 
+We will be using time as a metric as well as binary metrics for completion of the parts of the competition. 
+This includes ’does the rocker-bogie system works as intended’, ’will the rover stop when directed at an obstacle’, etc. 
+The details are defined in the minimum viable product section.
+
+\subsection{Minimum Viable Product}
+\label{subsec:mvp}
+The minimum viable product (MVP) has been defined by the basics of what can be called a 'functioning rover'. This includes features like suspension, motors, power and controls. The MVP is defined by the following features: 
+
+\begin{table}[hb!]
+    \caption{Minimum Viable Product}
+    \centering
+    {\renewcommand{\arraystretch}{1.5}
+    \begin{tabularx}{\linewidth}{LL} 
+    \hline
+    \textbf{Feature} & \textbf{Reason} \\
+    \hline
+    Rocker-bogie-like suspension & This is the NASA preferred rover suspension style. \\
+    Ability to control remotely (such as a gamepad) & Remote control is essential for rovers. \\
+    Ability to traverse terrain such as gravel, cement, and asphalt & Multi-terrain traversal is important for rovers. \\
+    Technical report on GitHub & This will allow for future teams to easily learn from this teams experience. \\
+    Rover hardware & Having a physical rover is important for a rover project as it allows for lessons to be learned in construction. \\ 
+    Present at Imagine RIT & This is essential for promotional value. \\
+    Self-driving & Partially autonomous navigation for future URC readiness. \\
+    \hline
+    \end{tabularx}
+    }
+\label{tab:mvp-one}
+\end{table}
+
+\subsection{Improvements}
+\label{subsec:improvements}
+The MVP is intended to be flexible in implementation depending on funding. 
+The purpose of the project is to gain experience in rovers and robotics and this can be done in a number of form factors.
+The size and materials are not defined because of this. 
+Should the funding allow for it there are many improvements that can be made beyond the MVP. 
+This allows for more experience to be gained, thus fulfilling the goals of the project.
+
+\begin{table}[ht!]
+    \caption{Improvements}
```

Comment:
Is funding required equal for each improvement? Which improvements are more expensive, which are more difficult? If you had infinite funding, would you work on all improvements in parallel or would you prioritize some over others?

I would also like to note that the completion of these improvements will depend on funding, available time, and team member experience/interests.

Reply:
+1

This is very important to consider @tjh2822